### PR TITLE
update examples of valid and invalid names

### PR DESCRIPTION
**Pull request by wjwwood**
addresses https://github.com/ros2/design/issues/96#issuecomment-260018135

### CODE REVIEW

File: articles/115_topic_and_service_name_mapping.md [hunk 1]
```diff
@@ -109,14 +109,15 @@ For example, these are valid names:
...
-| `foo/_/bar` | `rosservice:///foo` | `rostopic://foo/bar` |              |                  |
+| `foo/_bar`  | `rosservice:///foo` | `rostopic://foo/bar` |              |                  |
 
 But these are not valid names:
 
 | `123abc`   | `123`  | `__foo`  | `foo__bar` | `foo bar`   |
 | `foo__`    | ` `    | `foo_`   | `foo//bar` | `foo/`      |
 | `foo_/bar` | `~foo` | `foo~`   | `foo~/bar` | `foo/~bar`  |
 | `/_/bar`   | `_`    | `_/_bar` | `/~`       | `foo/~/bar` |
+| `foo/_bar` |        |          |            |             |
```
Comment (dhood) [inline articles/115_topic_and_service_name_mapping.md]: think you meant `foo/_/bar` here

Comment (mikaelarguedas) [inline articles/115_topic_and_service_name_mapping.md]: missed a `/` here maybe ? `foo/_bar` is supposed to be a valid name but `foo/_/bar` and `foo_/bar` aren't, right ?

**Comment by wjwwood** [comment]
Yup, to both of you. 😄

**Review by mikaelarguedas — APPROVED** [review]
lgtm

**Pull request merged**